packages/lib/pod/src/podUtil.ts

```diff
@@ -1,33 +1,4 @@
-import JSONBig from "json-bigint";
-import {
-  EDDSA_PUBKEY_TYPE_STRING,
-  PODEntries,
-  PODRawValue,
-  PODRawValueTuple,
-  PODValue,
-  PODValueTuple,
-  POD_CRYPTOGRAPHIC_MAX,
-  POD_CRYPTOGRAPHIC_MIN,
-  POD_INT_MAX,
-  POD_INT_MIN,
-  POD_NAME_REGEX,
-  POD_STRING_TYPE_REGEX,
-  POD_VALUE_STRING_TYPE_IDENTIFIER
-} from "./podTypes";
-
-// TODO(POD-P3): Decide if these utils should all be published outside
-// of the package, or only a subset.
-
-/**
- * Private keys are 32 bytes (any arbitrary bytes), represented as Base64 or
- * hexadecimal
- *
- * This regex matches any supported format, with match groups usable to
- * determine the format, in the order above.
- */
-export const PRIVATE_KEY_REGEX = new RegExp(
-  /^(?:([A-Za-z0-9+/]{43}=?)|([0-9A-Fa-f]{64}))$/
-);
+import { PODEntries, PODValue } from "./podTypes";
 
 /**
```

Comment:
It looks like you moved (or at least deleted) some of the _ENCODING_GROUPS constants, but not all?  They should all be in the same place.  I'd like them to move with the associated REGEXes since they're essentially metadata about the regex, but their format (and the CryptoBytesEncodingGroups type) are closely tied to the encoding/decoding functions, so I think it's okay for them to stay here too.

Reply:
I've moved the PRIVATE_KEY_ENCODING_GROUPS to podChecks.ts now.